components/IscnRegisterForm.vue

```diff
@@ -1069,6 +1088,9 @@ export default class IscnRegisterForm extends Vue {
   }
 
   async mounted() {
+    if (this.arweaveId) {
+      this.uploadArweaveIdList.push(this.arweaveId)
```

Comment:
Should reset `this.uploadArweaveIdList`? ie `this.uploadArweaveIdList = [this.arweaveId]`